src/trace-writer.js

```diff
@@ -244,16 +249,19 @@ TraceWriter.prototype.flushBuffer_ = function(projectId) {
  * @param {string} json The stringified json representation of the queued traces.
  */
 TraceWriter.prototype.publish_ = function(projectId, json) {
+  // TODO(ofrobots): assert.ok(this.config_.project), and stop accepting
+  // projectId as an argument.
   var that = this;
   var uri = 'https://cloudtrace.googleapis.com/v1/projects/' +
     projectId + '/traces';
 
-  this.request_({
+  var options = {
     method: 'PATCH',
     uri: uri,
     body: json,
     headers: headers
-  }, function(err, response, body) {
+  };
+  that.request(options, function(err, body, response) {
```

Comment:
Should this be `function(err, response, body)`? Both `this.request_` and `that.request` in the old and new revisions respectively are both assigned with a call to `util.makeAuthenticatedRequestFactory` (the latter in [`service.js`](https://github.com/GoogleCloudPlatform/google-cloud-node/blob/master/packages/common/src/service.js)).